tests/test_django_async_orm.py

```diff
@@ -198,9 +185,22 @@ async def test_async_prefetch_related(self):
     async def test_async_annotate(self):
         self.assertTrue(False, "Not Implemented")
 
-    @tag('dev')
+    @tag('ci')
     async def test_async_order_by(self):
-        self.assertTrue(False, "Not Implemented")
+        async def test_async_order_by_ascending(self):
+            qs = await TestModel.objects.async_all()
+            qs = await qs.async_order_by('name')
+            qs = await qs.async_first()
+            self.assertEqual(qs.name, "setup 1")
+
+        async def test_async_order_by_descending(self):
+            qs = await TestModel.objects.async_all()
+            qs = await qs.async_order_by('-name')
+            qs = await qs.async_first()
+            self.assertEqual(qs.name, "setup 2")
+
+        await test_async_order_by_ascending(self)
```

Comment:
would be better to split then into two different tests, always one assert per unit test